pkgs/development/libraries/gtk+/3.x.nix

```diff
@@ -21,12 +21,14 @@ stdenv.mkDerivation rec {
     sha256 = "1l45nd7ln2pnrf99vdki3l7an5wrzkbak11hnnj1w6r3fkm4xmv1";
   };
 
+  NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-lintl";
```

Comment:
Ugh I forgot about this, but `optionalString` will cause a linux rebuild too unfortunately. I think we need to instead say

```
NIX_LDFLAGS = if stdenv.isDarwin then "-lintl" else null;
```
to prevent the derivation from changing on linux. Is that right @shlevy?